e2e/windows/e2e-scenario.sh

```diff
@@ -95,6 +96,13 @@ fi
 log "Upload cse packages done"
 
 log "Scenario is $SCENARIO_NAME"
+export orchestratorVersion=$KUBERNETES_VERSION
+export windowsPackageURL="https://acs-mirror.azureedge.net/kubernetes/v${orchestratorVersion}/windowszip/v${orchestratorVersion}-1int.zip"
+if [[ -n "$WINDOWS_PACKAGE_URL" ]]; then
+    export windowsPackageURL="$WINDOWS_PACKAGE_URL"
```

Comment:
log for debug

Reply:
They are already in pipeline logs. Example:

`export orchestratorVersion=1.26.3
+ orchestratorVersion=1.26.3
+ export windowsPackageURL=https://acs-mirror.azureedge.net/kubernetes/v1.26.3/windowszip/v1.26.3-1int.zip
+ windowsPackageURL=https://acs-mirror.azureedge.net/kubernetes/v1.26.3/windowszip/v1.26.3-1int.zip`